views/index.pug

```diff
@@ -18,7 +18,7 @@ html(lang="en")
       .collapse.navbar-collapse#navbarContent
         ul.navbar-nav.navbar-left.mr-auto
           li.nav-item
-            a.nav-link#add-editor(href="javascript:;" title="Click or drag to desired destination") Editor
+            a.nav-link#add-editor(href="javascript:;" title="Click or drag to desired destination") Add Editor
           li.nav-item
             a.nav-link#add-diff(href="javascript:;" title="Click or drag to desired destination") Diff View
```

Comment:
For consistency, I'd suggest another change too

```suggestion
             a.nav-link#add-diff(href="javascript:;" title="Click or drag to desired destination") Add Diff View
```

Reply:
Sounds good. Made both three word phrases:
![image](https://user-images.githubusercontent.com/10781353/48368091-f6224700-e66f-11e8-89e6-fd3de138ed6f.png)